### PR TITLE
adding tribal boundaries

### PR DESCRIPTION
**Pull request by emma-nechamkin**
#1577

### CODE REVIEW

**Comment by emma-nechamkin** [comment]
requires update to fixtures -- will do in morning

**Comment by emma-nechamkin** [comment]
Need to (1) check addition of tracts / impact on population identified, (2) check proper tracts are included. This will take about an hour to do. I also must update the fixtures.

File: data/data-pipeline/data_pipeline/etl/sources/tribal_boundaries/etl.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+        The columns we need to know are:
+        - Percentage of Total Area of the Census Tract Covered by the American Indian Area: PERCENT_AREA
+        - Percentage of the 2010 Population Count of the Census Tract Covered by the American Indian Area: PERCENT_POP
+
+        There's also a field for share of housing units (PERCENT_HU) that we do not use.
+
+        For more information, see:
+            https://www.census.gov/programs-surveys/geography/technical-documentation/records-layout/2000-tract-to-aia-record-layout.html
+        """
```
Comment (esfoobar-usds) [inline data/data-pipeline/data_pipeline/etl/sources/tribal_boundaries/etl.py:106]: Really like this! 🚀

File: data/data-pipeline/data_pipeline/score/score_narwhal.py [hunk 2]
```diff
@@ -764,6 +764,16 @@ def _workforce_factor(self) -> bool:
...
+    # def _contains_tribal_area_or_population_factor(self):
+    #     return (
+    #         # strict greater than
+    #         self.df[field_names.PERCENT_AREA_FIELD_NAME]
+    #         > self.TRIBAL_THRESHOLD
+    #     ) | (
+    #         self.df[field_names.PERCENT_POPULATION_FIELD_NAME]
+    #         > self.TRIBAL_THRESHOLD
+    #     )
+
```
Comment (esfoobar-usds) [inline data/data-pipeline/data_pipeline/score/score_narwhal.py:776]: Not a fan of leaving commented code in a commit unless there's a good reason for it?

File: data/data-pipeline/data_pipeline/score/score_narwhal.py [hunk 3]
```diff
@@ -791,6 +801,10 @@ def add_columns(self) -> pd.DataFrame:
         self.df[field_names.N_WATER] = self._water_factor()
         self.df[field_names.N_HEALTH] = self._health_factor()
         self.df[field_names.N_WORKFORCE] = self._workforce_factor()
+        # # added
+        # self.df[
+        #     field_names.CONTAINS_TRIBAL_LAND_OR_POPULATION_ABOVE_THRESHOLD
+        # ] = self._contains_tribal_area_or_population_factor()
```
Comment (esfoobar-usds) [inline data/data-pipeline/data_pipeline/score/score_narwhal.py:807]: Same as above.

**Review by esfoobar-usds — APPROVED** [review]
🍏 with a small question

**Comment by emma-nechamkin** [comment]
it looks like we are not going to do this, so I am deleting this pr

**Pull request closed without merging**